addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java

```diff
@@ -29,8 +29,9 @@
 import oshi.util.EdidUtil;
 
 /**
- * This implementation of {@link SysteminfoInterface} is using the open source library OSHI to provide system
```

Comment:
Please make sure that you use the same formatting or revert this changes manually